sql/core/src/test/resources/sql-tests/inputs/string-functions.sql

```diff
@@ -108,6 +111,46 @@ SELECT lpad(x'57', 5, 'abc');
 SELECT rpad('abc', 5, x'57');
 SELECT rpad(x'57', 5, 'abc');
 
+-- Dsiable the lpad rpad binary breaking change
```

Comment:
it's probably an overkill to test the flag like this... I think the change in `StringExpressionsSuite` is good enough